### PR TITLE
zero out bitfields during subtraction when they end up empty

### PR DESCRIPTION
**Pull request by whyrusleeping**

### CODE REVIEW

File: rle/rleplus_writer.go [hunk 1]
```diff
@@ -2,14 +2,20 @@ package rlepluslazy
 
 import (
 	"encoding/binary"
+	"errors"
 )
 
+var ErrSameValRuns = errors.New("2 consecutive runs with the same value")
+
 func EncodeRuns(rit RunIterator, buf []byte) ([]byte, error) {
+	rit = newNormIter(rit)
```
Comment (magik6k) [inline rle/rleplus_writer.go:11]: Q: Do we ever care about trailing 0 runs when encoding?

Comment (Kubuxu) [reply rle/rleplus_writer.go:11]: No, we don't care about trailing zeros.

File: go.mod [hunk 2]
```diff
@@ -3,8 +3,9 @@ module github.com/filecoin-project/go-bitfield
 go 1.13
 
 require (
+	github.com/filecoin-project/specs-actors v0.4.0
```
Comment (Kubuxu) [inline go.mod]: go mod tidy?

**Review by magik6k — APPROVED** [review]

**Review by Kubuxu — APPROVED** [review]

**Comment by Kubuxu** [comment]
Rebased for CI to run.

**Pull request merged**